compiler/tests-compiler/error.ml

```diff
@@ -25,14 +25,18 @@ let normalize x =
   |> Str.global_replace (Str.regexp "node\\(.exe\\)?") "%{NODE}"
 
 let%expect_test "uncaugh error" =
-  let prog = {| let _ = raise Not_found |} in
+  let prog =
+    {|
+(* Ignore OCAMLRUNPARAM=b to be reproducible *)
+Printexc.record_backtrace false;;
```

Comment:
I think we should do something at the level of `compile_and_run` instead of repeating this logic everywhere.

Reply:
I have removed this logic from the tests.